validate.go

```diff
@@ -192,6 +194,48 @@ func checkProcess(process rspec.Process, rootfs string) {
 	}
 }
 
+func supportedMountTypes() (map[string]bool, error) {
+	supportedTypes := make(map[string]bool)
+	f, err := os.Open("/proc/filesystems")
```

Comment:
This probably needs to go behind a [`--host-checks`][1] (or [`--host-specific`][2] or some such) flag, since reading the local system's `/proc/filesystems` [is not portable][3].

[1]: https://github.com/opencontainers/ocitools/pull/49#issuecomment-216585596
[2]: https://github.com/opencontainers/ocitools/pull/49#issuecomment-220534023
[3]: https://github.com/opencontainers/ocitools/pull/49#issuecomment-217793768